examples/Using Machine Learning for Structural Identification.ipynb

```diff
@@ -19,7 +19,8 @@
     "import matplotlib.cm\n",
```

Comment:
<p>Are we filtering NumPy warnings because of NaNs or something? Nothing is raising the <code>FreudDeprecationWarning</code> anymore as far as I can tell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>